### PR TITLE
Run cargo clippy --fix

### PR DESCRIPTION
**Pull request by kovmarci86**
Run `cargo clippy --fix`

### CODE REVIEW

**Review by iamralpht — APPROVED** [review]
This change looks good to me (especially eliding the lifetimes). I'm surprised that clippy didn't merge the various Derives, but someone can clean that up by hand later (and overall `figma_schema.rs` is now much smaller by use of `#[default]` and `#[derive(Default)]`).

File: crates/figma_import/src/transform_flexbox.rs [hunk 1]
```diff
@@ -640,7 +615,7 @@ fn visit_node(
 
     // Check relative transform to see if there is a rotation.
     if let Some(transform) = node.relative_transform {
-        let parent_bounding_box = parent.map_or(None, |p| p.absolute_bounding_box);
+        let parent_bounding_box = parent.and_then(|p| p.absolute_bounding_box);
```
Comment (rylin8) [inline crates/figma_import/src/transform_flexbox.rs:618]: Didn't know about this one.

File: crates/figma_import/src/transform_flexbox.rs [hunk 2]
```diff
@@ -1105,7 +1078,7 @@ fn visit_node(
 
     // Convert any path data we have; we'll use it for non-frame types.
     let fill_paths = if let Some(fills) = &node.fill_geometry {
-        fills.iter().map(|figma_path| parse_path(figma_path)).flatten().collect()
+        fills.iter().filter_map(parse_path).collect()
```
Comment (rylin8) [inline crates/figma_import/src/transform_flexbox.rs:1081]: Didn't know about this either

**Comment by rylin8** [comment]
Learned some stuff from this!

**Review by rylin8 — APPROVED** [review]

**Review by iamralpht — APPROVED** [review]

**Pull request merged**